packages/circuits/provers/web.ts

```diff
@@ -0,0 +1,68 @@
+const KEY_URL = 'https://keys.unirep.io/2-beta-3/'
```

Comment:
I am afraid this will be hard to maintain
How about import version from `package.json`
```ts
import { version } from '../package.json'
```

and the key stored on server could also be named by version itself e.g. `2.0.0-beta-3`

Reply:
Good call, updated and updated docs as well.